test/specs/wrapper/find.spec.js

```diff
@@ -279,6 +280,68 @@ describeWithShallowAndMount('find', mountingMethod => {
     }
   )
 
+  it('have no better name right now, but this does not fail.........', () => {
+    const TestComponentToFind = {
+      render: h => h('div'),
+      name: 'test-component-to-find'
+    }
+    const TestComponent = {
+      template: `
+        <div>
+          <test-component-to-find/>
+          <test-component-to-find/>
+          <test-component-to-find/>
+        </div>'
+      `,
+      components: {
+        TestComponentToFind
+      },
+      name: 'test-component'
+    }
+
+    const wrapper = mountingMethod(TestComponent)
+
+    expect(
+      wrapper
+        .findAllComponents({ name: 'test-component-to-find' })
+        .at(0)
+        .exists()
+    ).toBe(true)
+  })
+
+  it('ok this test fails', async () => {
+    const TestComponentToFind = {
+      render: h => h('div'),
+      name: 'test-component-to-find'
+    }
+    const localVue = createLocalVue()
+    localVue.use(VueRouter)
+    const routes = [
+      {
+        path: '/a/b',
+        name: 'ab',
+        component: TestComponentToFind
+      }
+    ]
+    const router = new VueRouter({ routes })
+    // and this will not work with shallowMount because router view gets stubbed! How can I just test mount?
+    const wrapper = mountingMethod(
```

Comment:
`mountingMethod` is both `mount` and `shallowMount`. We run all tests for both. If you don't want to run the test for one, just import the one you want on line 2 from `packages/src/test-utils`.

Reply:
ok done that.